tests/integration/transformers/pii/test_anonymizer.py

```diff
@@ -69,6 +69,26 @@ def test_anonymizedfaker_with_nans():
         'id': [1, 2, 3, 4, 5],
     })
 
+    pd.testing.assert_frame_equal(transformed, expected_transformed)
+    assert len(reverse_transform['username']) == 5
+    assert reverse_transform['username'].isna().sum() == 2
+
+
+def test_anonymizedfaker_with_nans_missing_value_generation_none():
+    """End to end test settings missing_value_generation=None."""
+    data = pd.DataFrame({
+        'id': [1, 2, 3, 4, 5],
```

Comment:
Can we add a test for when it's random as well?

Reply:
I think it's the test `test_anonymizedfaker_with_nans()` just before because it's random by default.